templates/userprofile/profile_page.html

```diff
@@ -7,19 +7,25 @@
 <div class="d-flex flex-column align-items-center">
     <h1 style="color: white;">{{ user.username }}'s' Profile Page</h1>
```

Comment:
There seems to be an extra comma here, when I login I am seeing `susmitabhowmik's' Profile Page`